README.md

```diff
@@ -312,6 +312,38 @@ parser.push(manifest);
 parser.end();
 parser.manifest.segments[0].custom.vodTiming // #VOD-TIMING:1511816599485
 ```
+
+Custom parsers may also map data an existing tag type.
+```js
+const manifest = [
+    '#EXTM3U',
+    '#VOD-TIMING:wallclock=1511816599485',
+    '#EXTINF:8.0,',
+    'ex1.ts',
+    ''
+  ].join('\n');
+
+const parser = new m3u8Parser.Parser();
+parser.addTagMapper({
+  expression: /#VOD-TIMING/,
+  map(line) {
+    const regex = /#VOD-TIMING:(?:wallclock=([^,\n]+))/g;
+    const match = regex.exec(line);
+
+    if (match) {
+      const ISOdate = new Date(Number(match[1])).toISOString();
+      return `#EXT-X-PROGRAM-DATE-TIME:${ISOdate}`;
+    }
+
+    return line;
+  }
+});
+
+parser.push(manifest);
+parser.end();
+parser.manifest.segments[0].dateTimeObject // #VOD-TIMING:1511816599485
```

Comment:
should there be an option to do this at a segment level? Or do we get that for free?

If we get it for free, maybe the example should show a couple different segments to make it more clear.

Reply:
Yep we get it for free, all it does is map a line of text to another. I'll add another example to illustrate this.